wallet/wallet.go

```diff
@@ -3499,6 +3513,26 @@ func (w *Wallet) publishTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
 			backendError: err,
 		}
 
+	// Returned by bitcoind when a replacement transaction did not have
+	// enough fee.
+	// https://github.com/bitcoin/bitcoin/blob/9bf5768dd628b3a7c30dd42b5ed477a92c4d3540/src/validation.cpp#L830
+	// https://github.com/bitcoin/bitcoin/blob/9bf5768dd628b3a7c30dd42b5ed477a92c4d3540/src/validation.cpp#L894
+	// https://github.com/bitcoin/bitcoin/blob/9bf5768dd628b3a7c30dd42b5ed477a92c4d3540/src/validation.cpp#L904
+	case match(err, "insufficient fee"):
```

Comment:
What about when it is rejected for other reasons? As an example if it creates a chain of transactions that's deemed "too long".

Reply:
I intentionally didn't add it since I assume we won't be running into those situations, but agree that we might as well add it when at it. Done here: https://github.com/btcsuite/btcwallet/pull/647/commits/d131e20be1b1a16767893be343e3366e5544bbde